ContactList/ContactList/UpdateViewController.swift

```diff
@@ -67,44 +54,66 @@ class UpdateViewController: UIViewController {
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        
         if currentPersonForEditing == nil {
             currentPersonForEditing = Person()
+            navigationItem.rightBarButtonItems?.remove(at: 1)
```

Comment:
To avoid possible crash check `if navigationItem.rightBarButtonItems?.count > 1`